### ISSUE TITLE
Beautify and Improve the README 

### ISSUE THREAD

**Issue by SanjayMarreddi** [SanjayMarreddi/ML_Internship_Technocolabs#6]
Try to incorporate the following features :
1.  Add all the Appropriate README Labels
2.  Reduce the Content and enhance it with Images 
3.  Add a section for the Project Admin & Contributors
4.  Other things which you feel to be important 

Feel free to look at the Popular Project Repo's and incorporate the above points accordingly to our Project !!

**Comment by muthuannamalai12** [comment]
Can you assign me this issue

**Comment by SanjayMarreddi** [comment]
@muthuannamalai12 Sure, Go ahead. Good Luck !

**Comment by IshetaBansal** [comment]
Hey! 
I too would love to work on this issue. Do let me know if that's possible. Thank you :)

**Comment by SanjayMarreddi** [comment]
@IshetaBansal Yeah, You can work on it !  Improve the README's in the Folders. Good Luck !

**Comment by muthuannamalai12** [comment]
I have created a pull request. Kindly review it

**Comment by SanjayMarreddi** [comment]
@muthuannamalai12 Will review it soon and request changes if required.
Meanwhile You can start working on other Issues ! Good Luck

**Comment by muthuannamalai12** [comment]
Thank you. Sure I will start working on other issues

**Comment by akrish4** [comment]
Kindly do assign me @SanjayMarreddi

**Comment by SanjayMarreddi** [comment]
@akrish4 Done !

**Comment by himanshu007-creator** [comment]
i will like to work on this issue , if it stills need work.
I will add a banner + emojis + improve text

**Comment by SanjayMarreddi** [comment]
@himanshu007-creator Try to resolve the #30 or #34 . Good Luck !

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator Try to resolve the #30 or #34 . Good Luck !

#34  i fixes this issue , have a look here sir https://github.com/SanjayMarreddi/Facial-Expression-Recognition-Classifier-Model/pull/38
